packages/editor/src/components/rich-text/index.js

```diff
@@ -215,17 +212,29 @@ export class RichText extends Component {
 	 * @param {PasteEvent} event The paste event as triggered by TinyMCE.
 	 */
 	onPaste( event ) {
-		const dataTransfer =
-			event.clipboardData ||
-			event.dataTransfer ||
-			this.editor.getDoc().dataTransfer ||
-			// Removes the need for further `dataTransfer` checks.
-			{ getData: () => '' };
-
-		const { items = [], files = [], types = [] } = dataTransfer;
+		const clipboardData = event.clipboardData;
+		const { items = [], files = [] } = clipboardData;
 		const item = find( [ ...items, ...files ], ( { type } ) => /^image\/(?:jpe?g|png|gif)$/.test( type ) );
-		const plainText = dataTransfer.getData( 'text/plain' );
-		const HTML = dataTransfer.getData( 'text/html' );
+		let plainText = '';
+		let HTML = '';
+
+		try {
+			plainText = clipboardData.getData( 'text/plain' );
+			HTML = clipboardData.getData( 'text/html' );
+		// IE11 will throw an invalid argument error.
```

Comment:
This comment could be slightly expanded to say what causes it and why, eg:

```js
// IE11 will throw an invalid argument error when using `getData( 'text/html' )`
// because it doesn't support the HTML data type for clipboardData.
```

I also wonder if we should keep `plainText = clipboardData.getData( 'text/plain' );` out of the `try/catch` block because we don't expect it to fail, right?

Reply:
Cool, I'll update the comment. We don't expect plain text to fail? IE11 only supports the `Text`, so both would fail.

Reply:
Hmm, let me try moving it and testing in IE11 to see if it fails. I thought it accepted `text/plain`, but maybe I'm wrong!

Reply:
You're right, it does throw invalid argument, cheers. 😄